engine/sound/src/sound.cpp

```diff
@@ -1197,6 +1197,7 @@ namespace dmSound
                     }
                     else
                     {
+                        assert(false);
```

Comment:
Do we want an assert here or some warning/error log? In release this seems to be recoverable but the assert tells us that something is actually incorrect

Reply:
No, that's a debug assert for me, i just forgot it :)
I'll remove it now!